modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/layout-page-templates/master-pages/home/page-definition.json

```diff
@@ -164,308 +128,60 @@
-														"marginLeft": "2",
-														"marginTop": "4",
-														"paddingBottom": "2",
-														"paddingTop": "2",
-														"shadow": "",
-														"textAlign": "center"
-													},
-													"indexed": true,
-													"layout": {
-														"borderWidth": 0,
-														"contentDisplay": "FlexColumn",
-														"marginBottom": 2,
-														"marginLeft": 0,
-														"marginRight": 0,
-														"marginTop": 2,
-														"opacity": 100,
-														"paddingBottom": 0,
-														"paddingLeft": 0,
-														"paddingRight": 0,
-														"paddingTop": 0,
-														"widthType": "Fluid"
+											"fragmentFields": [
+												{
+													"id": "cp_footer_text",
+													"value": {
+														"fragmentLink": {
+														}
 													}
 												},
-												"pageElements": [
-													{
-														"definition": {
-															"fragment": {
-																"key": "custom-dialect-text"
-															},
-															"fragmentConfig": {
-																"textType": "h6"
-															},
-															"fragmentFields": [
-																{
-																	"id": "element-text",
-																	"value": {
-																		"fragmentLink": {
-																		},
-																		"text": {
-																			"value_i18n": {
-																				"en_US": "© 2023 Liferay Inc. All Rights Reserved&nbsp; &nbsp; &nbsp; |"
-																			}
-																		}
-																	}
-																}
-															],
-															"fragmentStyle": {
-																"borderRadius": "",
-																"fontFamily": "fontFamilySansSerif",
-																"fontSize": "0.875rem",
-																"marginLeft": "2",
-																"shadow": "",
-																"textAlign": "center"
-															},
-															"indexed": true
-														},
-														"type": "Fragment"
-													}
-												],
-												"type": "Section"
-											},
-											{
-												"definition": {
-													"fragmentLink": {
-														"href": {
-															"value_i18n": {
-																"en_US": ""
-															}
-														},
-														"target": "Self"
-													},
-													"fragmentStyle": {
-														"borderRadius": "",
-														"borderWidth": "0",
-														"marginLeft": "3",
-														"shadow": "",
-														"textAlign": ""
-													},
-													"indexed": true,
-													"layout": {
-														"borderWidth": 0,
-														"marginBottom": 0,
-														"marginLeft": 1,
-														"marginRight": 0,
-														"marginTop": 0,
-														"opacity": 100,
-														"paddingBottom": 0,
-														"paddingLeft": 0,
-														"paddingRight": 0,
-														"paddingTop": 0,
-														"widthType": "Fluid"
+												{
+													"id": "cp_footer_text_feedBack",
```

Comment:
`cp_footer_text_feedBack` should be `cp_footer_text_feedback` with a lowercase "b" in feedback